src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.name().toLowerCase();
+    private static final int FIGURES_NUMBER = 5;
+    private static final int MAX_RANDOM_NUMBER = 100;
+    private static Random random = new Random();
+    private static ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomCircle() {
```

Comment:
do you use these methods somewhere outside this class?

Reply:
No, I don't. You mean I should make them private?

Reply:
yes